### PR TITLE
Only load records if they are not already loaded

### PR DESCRIPTION
**Pull request by HSalmon**
Addresses https://github.com/loomio/loomio/issues/3737 and https://github.com/loomio/loomio/issues/3736

### CODE REVIEW

File: angular/core/models/discussion_model.coffee [hunk 1]
```diff
@@ -97,6 +97,9 @@ angular.module('loomioApp').factory 'DiscussionModel', (DraftableModel, AppConfi
       item = _.max @events(), (event) -> event.sequenceId or 0
       item.sequenceId
 
+    allEventsLoaded: ->
+      @recordStore.events.find(discussionId: @id).length == @itemsCount
```
Comment (gdpelican) [inline angular/core/models/discussion_model.coffee]: Ah, interesting, hadn't thought of it that way. This totally works though 🍏

**Comment by gdpelican** [comment]
Another couple nitpicks:

- Hitting 'ctrl+P' (as opposed to selecting print from the dropdown) opens the print window immediately without loading the discussion. Wonder if there's a way we can override that? (Don't worry about that for this PR)
- I'd love to see a loading spinner next to the loading entire discussion message, and maybe some additional messaging like 'this may take a little while depending on the size of the discussion); it looks like a good-sized discussion takes about 7 seconds or so to load fully.

**Comment by gdpelican** [comment]
Another thought to worry about later: We don't load all proposals for this, so if someone were printing out a discussion because they wanted a list of all the decisions which had been made, there's a chance they wouldn't get them.

**Comment by HSalmon** [comment]
screen shot 2016-10-03 at 12 21 44 pm

@gdpelican

**Comment by gdpelican** [comment]
🍏

**Pull request merged**